qdarkstyle/utils/__main__.py

```diff
@@ -50,28 +52,120 @@ def __init__(self, parser_args):
+    spec = importlib.util.spec_from_file_location(module_name, file_path)
+    module = importlib.util.module_from_spec(spec)
+    sys.modules[module_name] = module
+    spec.loader.exec_module(module)
+
+    return module
 
 
 def main():
     """Process QRC files."""
+    logging.basicConfig(level=logging.DEBUG)
     parser = argparse.ArgumentParser(description=__doc__,
                                      formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument('--qrc_dir',
-                        default=None,
-                        type=str,
-                        help="QRC file directory, relative to current directory.",)
-    parser.add_argument('--create',
-                        default='qtpy',
-                        choices=['pyqt5', 'pyqt6', 'pyside2', 'pyside6', 'qtpy', 'pyqtgraph', 'qt', 'qt5', 'all'],
-                        type=str,
-                        help="Choose which one would be generated.")
-    parser.add_argument('--watch', '-w',
-                        action='store_true',
-                        help="Watch for file changes.")
+    parser.add_argument(
+        "--base-path",
+        default=PACKAGE_PATH,
+        type=str,
+        help="Base directory where palette assest will be generated.",
+    )
+    parser.add_argument(
+        "--base-svg-path",
+        default=SVG_PATH,
+        type=str,
+        help="Base path were source .svg files are located.",
+    )
+    parser.add_argument(
+        "--palette-images",
+        default=False,
+        type=bool,
+        help="If the Palette preview files (`palette.svg` and `palette.png` "
+             "should be generated. The preview files will be generated following the "
+             "path provided through the `--palette-images-path` argument."
+    )
+    parser.add_argument(
+        "--palette-images-path",
+        default=IMAGES_PATH,
+        type=str,
+        help="Path where palette preview images (`palette.svg` and `palette.png`) "
+             "will be located.",
+    )
+    parser.add_argument(
+        "--resource-prefix",
+        default="qdarkstyle",
+        type=str,
+        help="Prefix used for this style.",
+    )
+    parser.add_argument(
+        "--style-prefix",
+        default="qss_icons",
+        type=str,
+        help="Prefix used in resources.",
+    )
+    parser.add_argument(
+        "--custom-palette-key-value",
+        nargs="*",
+        action=CustomPaletteParser,
+        help="List of `key=value` definitions for a custom palette. "
+             "Alternative to `--custom_palette_file` + "
+             "`--custom_palette_class_name` args.",
+    )
+    parser.add_argument(
+        "--custom-palette-file",
+        type=str,
+        help="Path to a Python file with the custom Palette subclass "
+             "definition. Alternative to `--custom_palette_key_value` arg. "
+             "Needs to be used alongside `--custom_palette_class_name` "
+             "to work.",
```

Comment:
```suggestion
        help="Path to a Python file with a custom Palette subclass "
        "definition. It needs to be used alongside "
        "`--custom_palette_class_name` to work.",
```

- Remove reference to `--custom_palette_key_value`.
- 79 columns.